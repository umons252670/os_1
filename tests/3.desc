Same as test_2
