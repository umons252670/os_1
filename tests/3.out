XV6_TEST_OUTPUT 150000
